interfaces/automated-testing/scd/scd.yaml

```diff
@@ -0,0 +1,41 @@
+openapi: 3.0.2
+info:
+  title: Strategic Conflict Detection Test Data Injection
+  version: 0.0.1
+  description: >-
+    This interface is implemented by every USS wishing to be tested by the automated testing framework.
+    The automated testing suite calls this interface to inject test data into the USS system under test.
+
+components:
+  schemas:
+    StatusResponse:
+      type: object
+      required:
+        - status
+      properties:
+        status:
+          description: >-
+            The status of the USS automated testing interface.
+
+            - `starting`: the USS is starting and the automated test driver should wait before sending requests.
+
+            - `ready`: the USS is ready to receive test requests.
+          type: string
+          enum: [starting, ready]
+          example: ready
+
+paths:
+  /uss/v1/test/scd/status:
```

Comment:
Thank you very much for the background information and the proposal. I think it would be valuable to capture this information but I am not sure yet how.